prelude/prelude.purs

```diff
@@ -120,6 +120,9 @@ module Prelude
   class Functor f where
     (<$>) :: forall a b. (a -> b) -> f a -> f b
 
+  fmap :: forall f a b. (Functor f) => (a -> b) -> f a -> f b
+  fmap = (<$>)
```

Comment:
:+1: for named and `map`.

It seems like there may be an opportunity coming up for extreme breaking of things with good reason - if we can get rewrite rules to where we want and entirely externalise `Prelude`, then things like `Functor` and `Monad` should be moved into their own modules under `Control` anyway in my opinion. If we go ahead with that, re-naming the operations at the same time seems like the time to do it.